src/test/java/pages/HomePage.java

```diff
@@ -1,20 +1,51 @@
 package pages;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
 
 public class HomePage extends BasePage {
 
-    By userAvatarIcon = By.cssSelector("img.avatar");
+    //By userAvatarIcon = By.cssSelector("img.avatar");
+    @FindBy(css = "img.avatar")
+    WebElement userAvatarIcon;
+    //By firstPlaylist = By.cssSelector(".playlist:nth-child(3)");
+    @FindBy(css = ".playlist:nth-child(3)")
+    WebElement firstPlaylist;
+    //By playlistInputField = By.cssSelector("input[name='name']");
+    @FindBy(css = "input[name='name']")
+    WebElement playlistInputField;
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement use
```